flutter/android/src/main/kotlin/io/sentry/flutter/SentryFlutterPlugin.kt

```diff
@@ -354,20 +354,25 @@ class SentryFlutterPlugin : FlutterPlugin, MethodCallHandler, ActivityAware {
   }
 
   private fun captureEnvelope(call: MethodCall, result: Result) {
+    if (!Sentry.isEnabled()) {
```

Comment:
@brustolin or @philipphofmann do we need to do this guard for iOS before calling `PrivateSentrySDKOnly.capture`? or does `capture` check `isEnabled` internally already?

Reply:
If the SDK in the native side is disabled then we don't capture the envelope.
But I would keep this check in the dart side as well.

Reply:
`capture` already calls `SentryClient#isDisabled` so that's all we need, to need to double guard in this case, thanks.